src/collections/pages/LabelListPage/LabelListPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Box } from '@material-ui/core';
+import { Button, HandleApiResponse } from '../../../_shared/components';
+import { LabelListCard } from '../../components';
+import { Label, useLabelsQuery } from '../../../api/generatedTypes';
+
+/**
+ * Label List Page
+ */
+export const LabelListPage = (): JSX.Element => {
+  const { loading, error, data } = useLabelsQuery({});
+
+  return (
+    <>
+      <Box display="flex">
+        <Box flexGrow={1} alignSelf="center">
+          <h1>Labels</h1>
+        </Box>
+        <Box alignSelf="center">
+          <Button
+            component={Link}
+            to="/collections/labels/add/"
```

Comment:
just to verify - this page doesn't exist yet, right?

Reply:
Yes!

Reply:
Also, I think we have a custom button component that you can use and it comes with all the styles. Could be worth checking and making the code less redundant given our pending mui5 chore?

Reply:
@Herraj correct me if I am wrong but I think this is using the custom `Button` component from `_shared/components`?

Reply:
oh whoops, yes you are right. All good!